tests/foreman/cli/test_organization.py

```diff
@@ -139,7 +139,7 @@ def test_positive_CRD(self):
 
         # List
         result = Org.list({'search': 'name=%s' % org['name']})
-        self.assertTrue(len(result) > 1)
+        self.assertTrue(len(result) > 0)
```

Comment:
Code looks fine , however would like know why that is require to change from `1` to `0` ? is that due to product change ?

Reply:
@vijay8451 This was added in #7046. Previously, this assertion was part of `test_positive_list`, where it was `self.assertTrue(len(result_list) > 0)`. I think `0` was changed to `1` by mistake, not exactly sure how.

@pondrejk Perhaps you could use the same idiom that you use below, 
```
self.assertTrue(len(result), 1)
```